docs/source/models/cause.rst

```diff
@@ -337,14 +336,48 @@ Mortality Impacts
 Cause-Specific Mortality
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
+The cause-specific mortality rate is the mortality rate from a specified cause for a population. 
+The numerator is the number of deaths attributed to a specific cause. The denominator remains the 
+size of the population. In GBD, cause-specific estimates by country, year, age, sex.
+
+  For example, 112 measles deaths in country A during calendar year 2016. 1010,623 estimated 2016 mid-year 
+  population. 11.08 measles deaths per 100,000 population person-years at risk during calendar year 
+  2016 in country A. 
+
 Excess Mortality
 ^^^^^^^^^^^^^^^^
 
+The excess mortality is obtained by subtracting the expected mortality from the observed mortality.
+The excess mortality can be interpreted as the mortality directly or indirectly related to the disease of 
+interest, if one excludes all other causes of death. It is the extra number of deaths observed, beyond that 
+expected, per person-years.   
+
+  For instance, an excess mortality rate of 40 deaths per 100 person-years means that there are 40 additional 
+  deaths per 100 person-years in excess of those expected. These deaths are supposed to be related to the disease 
+  of interest.
+
 Morbidity Impacts
 +++++++++++++++++
 
 Disability Weights
 ^^^^^^^^^^^^^^^^^^
```

Comment:
I think we should add in a paragraph that defines disability weights with examples at the beginning of this section. It could look something like this:

Disability weights are measures that reflect the severity of the disability associated with a specific cause 
and range from 0 (perfect health) to 1 (equivalent to death). In other words, disability weights represent 
the proportion of full health lost when an individual is living with a given condition.

   For example, mild anemia has a disability weight of 0.004, whereas severe anemia has a disability weight of 
   0.149. In contrast, severe multiple sclerosis has a much higher disability weight of 0.719. As seen in these
   examples, the more severe the cause, the disability weight, and the greater the proportion of full health 
   lost due to the condition.

Reply:
Thank you Ali, I have added the paragraph.